### DISCUSSION TITLE
How do I get dotnet build to error out with <Features>strict</Features>

### DISCUSSION THREAD

**Discussion by anishkny** [dotnet/roslyn#69728] [Q&A]
Hello,

I am sorry if this is not the correct forum.

I am trying to get `dotnet build <project>` to fail if there are any warnings. I have enabled strict mode using `<Features>strict</Features>` in my project. Currently, it displays warnings but does not error out with a non-zero exit code.

Thanks!

**Comment by jasonmalinowski** [comment]
strict means something else; you're looking for [TreatWarningsAsErrors](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/errors-warnings#treatwarningsaserrors).
